libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/utils/ActiveLearningUtils.java

```diff
@@ -22,27 +22,35 @@
      */
     private static final Float MAX_LOW_SCORE_VARIATION_MULTIPLIER = 1.0f;
 
-    private static Float MAXIMUM_SCORE_FOR_LOW_SCORE_VARIATION = 95.0f;
+    private static final Integer PERCENTAGE_DIVISOR = 100;
 
-    private static Float MINIMUM_SCORE_FOR_LOW_SCORE_VARIATION = 20.0f;
+    private static final Float MAXIMUM_SCORE_VARIATION = 95.0F;
+
+    private static final Float MINIMUM_SCORE_VARIATION = 20.0F;
+
+    private static Float maximumScoreForLowScoreVariation = MAXIMUM_SCORE_VARIATION;
+
+    private static Float minimumScoreForLowScoreVariation = MINIMUM_SCORE_VARIATION;
```

Comment:
This is weird, why we can't assign the values directly in `maximumScoreForLowScoreVariation` and `minimumScoreForLowScoreVariation` converting them as UPPER_CASE?

Reply:
This change will update the entire file

Reply:
Here is all because the linters errors. The changes of the name of the variables and also the assignment of the values in static final variables (magic numbers errors)

Reply:
But why we can change `private static Float maximumScoreForLowScoreVariation = MAXIMUM_SCORE_VARIATION;` to `private static final Float MAXIMUM_SCORE_FOR_LOW_SCORE_VARIATION = 95.0f;`?

Reply:
This `private static final Float MAXIMUM_SCORE_FOR_LOW_SCORE_VARIATION = 95.0f; ` cannot be done because that variable has an assignment in the class, thus cannot be final

Reply:
Ack